documentation/crds_users_guide/source/programmatic_interface.rst

```diff
@@ -392,3 +392,18 @@ This will validate the submission by making sure all of the fields that need val
 have them and by ensuring that there is at least 1 file to submit before performing
 the submission.
 
+The return value of `submit()` includes error and warning counts, as well as
+a link to the Review/Confirm page for the submission::
+
+    result.error_count
+    result.warning_count
+    result.ready_url
+
+The `open_ready_url()` function will attempt to open the Review/Confirm page in
+your system's default browser::
+
+    result.open_ready_url()
+
+Note that the page's *confirm*, *cancel*, and *force* buttons will not be available
```

Comment:
Seems worth noting, I spent an embarrassing amount of time puzzling over why those buttons were behaving "inconsistently".